modules/get-started/pages/licensing.adoc

```diff
@@ -0,0 +1,25 @@
+= Enterprise Licensing 
+:description: Learn how to purchase and apply an Enterprise Edition license.
+
+Anyone can use the certified and community connectors, but you need an Enterprise Edition license to use Enterprise connectors. 
+
+== Purchase an Enterprise Edition license
+
+You can purchase and download a license from the http://redpanda.com/try-enterprise[Redpanda website^].
+
+== Apply a license key to Redpanda Connect
+
+Once you have downloaded a license key, you can apply it in the following ways:
+
+. Save the license key to any local directory and run the following command:
+
++
+```bash
+rpk connect run --redpanda-license
```

Comment:
the `--redpanda-license` flag needs the actual contents of the license to work, so it'd look something like this:

```
rpk connect run --redpanda-license "eyJ2ZXJzaW9uIjogMCwgIm9yZyI6ICJtZW93IiwgIn..." ./foo.yaml
```

Reply:
Updated. But when I tested and added a valid license key, I couldn't see any message about the expiry date.